recipes/ecctl/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "ecctl" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: go-{{ name|lower }}
```

Comment:
I'm not in the go team but I don't recall we using prefixes for go packages. Ping @conda-forge/core here in case I'm outdated.

Reply:
@ocefpaf I removed the go- prefix from the name and changed the version number to avoid hardcoded values.